google/cloud/storage/bucket_metadata_test.cc

```diff
@@ -86,6 +86,9 @@ BucketMetadata CreateBucketMetadataForTest() {
         },
         "etag": "AYX="
       }],
+      "encryption": {
+        "defaultKmsKeyName": "/a/key/name/should/go/here"
```

Comment:
I know this is just dummy data, but aside from having several slashes, this isn't close to a valid name.  Can we use this value instead, here and below?

```
projects/test-project-name/locations/us-central1/keyRings/test-keyring-name/cryptoKeys/test-key-name
```

Reply:
Done.  Ugh, who designed those.